pkg/runtime/finch.go

```diff
@@ -59,7 +59,7 @@ func (f *FinchRuntime) ContainerWithPort(ctx context.Context, name string, port
 		return false, err
 	}
 
-	var containers []Conainer
+	var containers []Container
```

Comment:
same as above

Reply:
Looks like these are all contained in this file only so I don't expect to run into any issues.